libs/features/components/banners/src/lib/components/team-member/team-member.component.ts

```diff
@@ -0,0 +1,42 @@
mnqxzSLAyjJpV8O5rjtHbyTNMVFVuvBprFSCu0gO7AdDRxwVNVeuxCSyRskMjjceRBo4q2FzXoygKBSOOQexd6VWC/ZXfzPM/P5U3uwp23zTwDjwqfSuw4iYeLmc5+fxwffTwy5yPn53FDkhBuduvr8imGc8k3PlQYbLewsLi/fu7SMyHG+Mcv6fhV4ex+qJCZGEKnGytIcn+HoKN7K2w021a6uT3bopySevXHz0pthc6trFyZHbu7UP4MfakGeZ8hXFPNPZ69I9XF4uNxTm+WZ6OO3jlK6hdrbENwtGBl1Pljp/SriLSorwrFpMMjy4Dd5OwAYY3xv6ir3XNFF/Ktza/RJrqKIIyfYkPrnqRvttUejTW2nys1/K0DJ9WqBSSzE8hjnsCdulUxuORXfJzZVLDJx14+zOXml31rcGCa2kMn+ReIH2EVXuvC2D0r0S47TWVkpNtPcvKwZVaN1Yj3enr5ViprWG7uEFhefSndcu0xCsz5OdvdTtVySi74K+lABNPuLeW3kMc0bRuOjCo96SxnEVlWoigzT8GlIAFFMDiQmInlTTb55iiUYCpCy+VHYGgD9FHlSfRF8qNMgpvEMUbYuoCbZM8q6iiwrqNsFIvpZH7sYBqEtlPKnXl7DFEcEFuS43FZ28vGPhjdiScszVCMWzqk0izutRht8pxEt/lGcUBPq0EsHdlW+1kkiqyVmJHG2WO59BQ58W55eVdEIJcnNkk5KgprqMHwrk7VC1wx2UAVEdwD0pKqSoVmJ5nNGaPD396iN9geJj7KDNWGitwTvJ+yopZdD4/2RtI7F75EtcsLWNuMoG2ZumfT0q+luItGswHieSVxhI0G9U2i6kEtI8j6zPxNaSKFbgl52Dv1A6V5WVu+ej3caSRQ6XZal3s11JccCO3EqcIPdj29aznaLVJh31qTubp+PJ34eFBjPPBrUarNqcrfR7W17qAN9t5ApI9APzrC9rklTVeKReFmjUnfOTyz8AKt4z/Mj5y/h/xgHeGeXdiE4WLKhwFUdB5eXvFdICtqrHi4zwsSDhVHQY8+vpQ/GwWRR99OHPvB/KipAbnLCXwRqM8RAA8sV3uzx0i5/tCa5ijtpHeZIhxwPJ9rgP3T54IIz6U5COoqXQdJlvoHECOZYbZXXxbHibIHpsQa4wtG7RyoVdTgg9DXPKk6R1R2atg8kgJwKYckUSsQ4jsKeId6FgpgQVhTipAzRTxGuSLjGPKtZtWAZJbBp3D1olkRDvsaQqpU4NNYtALtvXU54SW2rqbgSiO+nMecMGcn4eQoOVn7woHwq7M+OfnRV2kduxeaQu33UAxiq4l5zhF2G5HkKMf6NLsSWVSOGIYXqerVFnIxTyoUYI8XU03FURNnE+XKkrq7rRAdRumHDP7qCNG6cPtN6gUs+h8f7IvLR34wM7A9K0Wk6hPFOpZ2aMncVQWakkkDOav9LgyRn7Od687Ke5gXBY65rK2Vj9IihDuTwqGbGT8+Vee9pbt7u5RpiDKqfWFeQJ5D4Ud21umGpolu5EUJyME7P1rMklmLEkljkk8ya6PHwpJSOHy/Iu8Y2jbdpJI4kNvG/A31bHnvzBHUe2oUt3lXijGR1ozTJ0tr9Gk/VJ1bpXTKXHByY4JySkehdlrK3tYlaS4/6uc8Ui5IJJ6YB5bVB2rihivVkizxSL49uorL6Rqcf046hdznv2PhA6DoMCre/v0nv7ee7bK3K9yNyFRRk8R9c49m9cUcc/ZbO7Llx+pKIMqhhnG9MLFT5066HdSMnVdjQ4lzsapRy2TPOOHBFRwzYJ351FIqscA1G0TKBg0aQLYS4DOM0kqqjAAihiJCeeffTJFbIJNGgbErkcXOuoctk0tNQllPO/eOWO5PWpoW4LQY24nJb1AxgU+KECETTYVcZC+dDTTcWRGPDT3fCNWvLI5PtnfrtSE4O1KctjPPpSVQmJnBrh511d0rAONXGjRgsAUyD07tWz8RVOAWOBzNaLSFltySid4SM8PIJ60RZMtGuLfT3QPD4pHwqg9PPG/5Vci9tks2lgyzhSwi5MxHQDrWNZJINRlNzOc4JSQjwnr64qW24pZ4z37oUIZWnIZDvtkYGK554McnZ1Y/KzQjXwBSWWo3csksttIS8hYj2nP50DPBLZ3DRToUbHI9RWz026lubif6XAEKAAcJ6+h+8NjWW1tklv5mDDiVuHBO+OlVjfRGXPJFbz8GcKqnrgc6ZeMkioUztzzvUK4OzE7+lPeNe78PGx55yMAUaBbZNZDDKxAxmrednv7MQQwO8kfjDxnZfUn41W6cqPEQ0KMxXwMy8W/ngmri1vmgEReTOIuF4FACn2Dz91ZmRLbuL+yLOOG8g8MgH3sDn7McqGxknFOuphayDVLUExkIkiHb+HmOXlSlQLghGyh3U/tKdwfhipyVclYyshVTxbmpUy7cNLNbsPEu9Di5ERwRvSBvkLEPC2SaZIFOcmmd/wAYyKEuZiAQMiskwNoVmUEiuoYS7bjeupxSLUrgSNGsZ8AHIUEOdK/MeykFUgqQs5bOxx+7joKb1pxpOtEA2lrjSUQEkOe8yqliPStKLw6c0YmjIMgBV+a5wDuDvyNVGkcmqbW/7tbfP/bSj8CPlhUGY9R72RQVkZjJjcBTyBHPHtFXh0WMwk2ihzKAGjVwRw55jPs86o4/8Ms/9RrT2H6s/wCn/wCGpMg+PsCsYreCykSOR2SN+Euq5zITvj2bD41jdVXh1G4HEW8Z3xitVpH+HJ+9/I1ldR/vk/7xqy7MDqedSidTGEkXHDsCMjIqEda7pTARZ6cInh+t74cJweA7e+riO3tkVDEAN9idziqTTf1U3sq7T+7wf+v4igZk9/AJ9Pu4RvmMuMftLvt7aqY3dbe1YAbwKDj3/lirqP8AUv8AuW/Cqn/wrX9yv4Uk+hoBUV2vBhjvigpu6kk4mNRLzaoH50iQ7Yazqq+EihpZA+xxmmryqH79FGkx7KM7UlOrqYU//9k=',
```

Comment:
This image address is too long, I'm sure you can find a shorter one on unsplash or pexels etc